build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
Commit message should describe the commit contents comprehensively. Updating the gradle tools version should not be part of this commit.